### PR TITLE
First pass on the Node-Value Inspector [#95934234]

### PR DESCRIPTION
**Pull request by knowuh**
This is a First pass on the Node-Value inspector request.  The Node-Value inspector will display the initial value for a node when the simulation is started.  The Node-Value inspector allows students to set the initial value using a slider or text field.

Full details in the PT story:
  https://www.pivotaltracker.com/story/show/95934234

Followup commits will connect nodes to these values.  I am putting in a PR now so that it doesn't get too long.  A similar PR is in the works for the relationship inspector.

@sfentress could you take a look?

### CODE REVIEW

**Comment by knowuh** [comment]
Maybe @scytacki you could take a look at this?  I will have aaron or take a look my other WIP branch.

**Comment by dougmartin** [comment]
@knowuh @scytacki - I can look at it also this afternoon if needed.

**Comment by scytacki** [comment]
Doug that would great, thanks.

--
from my phone

File: src/code/views/node-value-inspector-view.coffee [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+  render: ->
+    (div {className: 'value-inspector'},
+      (div {className: 'inspector-content group'},
+        (span {className: 'full'},
+          (label {className: 'right'}, tr "~NODE-VALUE-EDIT.INITIAL-VALUE")
+          (input {
+            className: 'left'
+            type: "number",
+            min: "#{@props.min}",
```
Comment (dougmartin) [inline src/code/views/node-value-inspector-view.coffee]: I think you can drop the quotes here.

**Comment by dougmartin** [comment]
Looks good :+1:

**Pull request merged**